### ISSUE TITLE
Visualizing pytorch Emited NVTX with Tensorboard Profiler

### ISSUE THREAD

**Issue by jmoork** [pytorch/kineto#787]
There are important Pytorch Specific markers generated during the training loop either using NVTX or other means from pytorch lightning.

It would be useful to show the time line view with execution time for different pytorch training sections of the code as shown in the attached image. The current trace view is quite detailed and goes down one-step further with detailed cuda kernels etc. It would be useful to have Pytorch Execution specific trace view that is easy to understand and intuit to find any synchronization or communication bottlenecks in the training loop and also compare different execution time for different executing region of the training/validation code (like forward, loss, data-loading etc).

![Screenshot 2023-07-19 at 5 05 22 PM](https://github.com/pytorch/kineto/assets/12772531/0061929b-520b-4572-9be4-4f86c1ec8444)

**Comment by alphaRGB** [comment]
Hello, how you 

> There are important Pytorch Specific markers generated during the training loop either using NVTX or other means from pytorch lightning.
> 
> It would be useful to show the time line view with execution time for different pytorch training sections of the code as shown in the attached image. The current trace view is quite detailed and goes down one-step further with detailed cuda kernels etc. It would be useful to have Pytorch Execution specific trace view that is easy to understand and intuit to find any synchronization or communication bottlenecks in the training loop and also compare different execution time for different executing region of the training/validation code (like forward, loss, data-loading etc).
> 
> ![Screenshot 2023-07-19 at 5 05 22 PM](https://user-images.githubusercontent.com/12772531/254738222-0061929b-520b-4572-9be4-4f86c1ec8444.png)

Hello, how you implement visualize the training step in NVIDIA Nsight System.  By manually adding nvtx.push_range/pop_range() ?  This can be a flexibility way to show.
But, if we want to visualize the operation (like aten::conv/mm/softmax), we can't inject the nvtx mark into the C++ kernel. Nsight can visualize the CUDA kernels/cudaRuntime function, but cannot see PyTorch aten operations, is there good way?

**Comment by jon-chuang** [comment]
Hello, is NVTX captured by kineto atm?
